sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -210,7 +213,7 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
         currentAssessmentInitialValues: {
           assessmentWorkflowId: "some fake id",
           assessmentDate: new Date(),
-          studentAssessmentStatus: StudentAssessmentStatus.InProgress,
+          studentAssessmentStatus: StudentAssessmentStatus.Completed,
```

Comment:
Is the status required to be `Completed` here?

Reply:
Yes, it is required and yes, the idempotency relies on it now.
The E2E is testing a scenario where the workflow data is already set, which means that an `InProgress` status was not accurate.